ParabolInc/parabol
# feat(jira-server): push a task to Jira Server integration

Resolves #5810 #5994 #6238


**How to test:**

- Add Jira Server integration
- Create a task in parabol
- Push task to Jira Server using push to integration menu
- See the task was created on Jira Server
- Add another member to a parabol team
- Login as a new member
- Create a task
- Login as a previous member
- Run the above query for the other member task
- See the task was created on Jira Server and a comment added
- Edit task in Jira Server
- Open parabol board, see the task is loaded correctly


![image](https://user-images.githubusercontent.com/466991/155158609-99a44731-9944-4dda-a2df-ff7bb211a6e6.png)
